webserver/__init__.py

```diff
@@ -1,68 +1,83 @@
 import os
 import sys
+
+from flask.config import Config
+
 sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+basepath = os.path.dirname(os.path.realpath(__file__))
 
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
 import logging
 import logging.handlers as handlers
 
-from helpers.StreckennetzSteffi import StreckennetzSteffi
-from database.engine import DB_CONNECT_STRING
+# Create app with changed paths  https://stackoverflow.com/a/42791810
+app = Flask(__name__, instance_relative_config=True,
+        template_folder='website', static_folder='website/', static_url_path='')
 
-basepath = os.path.dirname(os.path.realpath(__file__))
-
-streckennetz = StreckennetzSteffi(prefer_cache=True)
-from webserver.predictor import Predictor
-pred = Predictor()
-db = SQLAlchemy()
+logHandler = handlers.TimedRotatingFileHandler(basepath + '/logs/website.log', when='midnight', backupCount=100)
+formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+logHandler.setFormatter(formatter)
+logHandler.setLevel(logging.INFO)
+app.logger.addHandler(logHandler)
+app.logger.setLevel(logging.INFO)
+logger = app.logger
 
+#Print our cool logo
+from helpers.fancy_print_tcp import TCP_SIGN
 
-def create_app():
```

Comment:
I don't get why you don't want to use an application factory. It is recommended by [Flask](https://flask.palletsprojects.com/en/1.1.x/tutorial/factory/) and it makes it much much much much cleaner, more performant and saver. For example, when reimporting Stuff from this file, it does not create an app each time.

Reply:
But it doesn't recreate this stuff, or at least i've never seen it do that before... and this file should only be called when creating the webserver.
The other way makes it much cleaner in my perspective.

But if there is an practical choice, i can reconsider this

Reply:
And it allows you to use the normal `flask run` command, which if I rember correctly didn't work with that function

Reply:
`flask run` still works if you cd into `webserver/` or set `FLASK_APP`. So no change there.

Reply:
The thing is, it is recommended in all tutorials and I don't see any downside. It may not be needed but if we want to use several workers with gunicorn it seems to be more stable to me.